### DISCUSSION TITLE
几个建议，如果可以的话。节点管理。

### DISCUSSION THREAD

**Discussion by wengzhenquan** [2dust/v2rayNG#3480] [General]
v2rayNG是非常好的APP，在我看来比clash好太多了，但现在市面上像v2ray这样的貌似是唯一一个，我找了很久都找不到。
在我看来，v2最大的优点就是可以对多个订阅进行混合对比，以便于用户选用。
clash仅仅只能对单个订阅进行应用，管理节点非常不便。
我希望2能够尽量发挥这方面的优势，所以提几个建议，当然，如果开发者觉得没有意义，可以无视。

由于本人日常除了自己使用，也会整合筛选，保证质量和可用性的节点进行分享。

建议：
1、订阅分组显示当前剩余节点数量。

2、支持单独节点测试真链接操作。
（好像目前只有批量操作，只有PC端的v2rayN可以单独节点。但我发现经常-1的节点也是可用的，重新测试又没问题了，但“全部测试”时间太久，我需要单独测试，生成延迟数值，以便于重新排序。）

3、多选操作。
（有时候需要对指点的多个节点进行删除、测试真链接、分享等操作。关于分享操作，我可以自定义选用几十个优质节点，进行分享，而不是全部导出。）

另外，提一个貌似有点问题之处。
当我切换到某个订阅分组下，点击测试真链接事，是测试本分组下的节点真链接，这个逻辑没问题。但在同一个分组下，点击“删除全部配置”，导致其它分组也全删掉了，不是应该跟测试真链接的逻辑一样才对吗？
以至于现在，我需要单独删除分组内的全部节点，必须连整个分组都删除掉。

以上是我的所有建议，祝愿V2能够越做越好，发挥自己的优势，不会被clash类型的智障式“一键连接”打败。

**Comment by 2dust** [comment]
分组的一些问题在此已经处理 https://github.com/2dust/v2rayNG/releases/tag/1.8.37

**Comment by 2dust** [comment]
你提到的多选等细节操作，在手机上这样操作会很难用，PC有鼠标和键盘。
单个测试这其实很简单，但是界面不好做，现在一行已经有三个按钮了，再增加几个不好点击。
所以需要大家给出具体的操作建议，而不是开发者闭门

**Reply by wengzhenquan** [reply]
就像我们手机相册一样，长按列表里的某张照片之后，可进入多选操作如何？（列表下方跳出几个在多选操作下支持的按钮）
至于单个节点的“测试真链接”，我们从多选状态下，点击原来右上角列表下的“测试真链接”，就能支持任意数量的“测试真链接”操作了。

**Reply by 2dust** [reply]
10来个你可以多选，几百个就很难受了。
如果节点不经常变的情况下，把这几个节点拷贝到新组更好。

**Reply by wengzhenquan** [reply]
可以的，咱们先做到支持多选模式下的操作，至于以后能不能想到更好的方案，以后再一步步升级。
这样，多选下的删除，也能够批量删除了。

目前“删除所有配置”，会把所有分组上的节点全删掉呀，就像我开始描述的那样。

**Reply by wengzhenquan** [reply]
其实右上角三个点里的菜单已经有很多功能了，包括多节点分享（导出全部配置到剪切板）、删除全部配置等。
只需要增加一个多选操作，自定义勾选哪些节点之后，复用右上角那一列菜单，就足够了。只不过应用范围是作用在用户选择的那些节点上

**Reply by luanyaolingwu** [reply]
话说能不能顺带给手动添加的节点更换分组的功能, 就像桌面版那样 (主要是有时候手残批量导入错分组了混一起很难受)
